2996_reflection/reflection.md

```diff
@@ -4568,6 +4568,8 @@ After the example following the definition of _manifestly constant-evaluated_, i
 
 [Special rules concerning reachability apply to synthesized points ([module.reach]{.sref}).]{.note13}
 
+An injected declaration whose name is an identifier reserved by the implementation is ill-formed; no diagnostic is required.
```

Comment:
This is not correct, we're not talking about the name of the _injected declaration_. We need to talk about the names of the data members of the injected declaration.

Reply:
Good point. Changed to:

> An injected declaration that introduces a name reserved by the implementation is ill-formed; no diagnostic is required.

Which I think works.

Reply:
I don't think that's right? Does `struct C { int x; };` introduce `x`?

Reply:
Trying again.